config/config.go

```diff
@@ -231,6 +231,14 @@ func (s *Struct) EEPROMPackageOrDefault() string {
 	return *s.EEPROMPackage
 }
 
+func (s *Struct) SudoOrDefault() string {
```

Comment:
Instead of defining this method on the config.Struct, you can directly define it in config.InternalCompatibilityFlags. That makes it a little clearer when to remove the method again :) (When we get rid of InternalCompatibilityFlags).

Reply:
Makes sense! Updated :)